components/text_countdown.js

```diff
@@ -52,11 +62,11 @@ export class TextCountdown extends Component {
         const seconds = Math.round((effectiveDelta % 60000) / 1000)
 
         const components = []
-        if (days) components.push(this.formats.days.replace("%@", days))
-        if (hours) components.push(this.formats.hours.replace("%@", hours))
-        if (minutes) components.push(this.formats.minutes.replace("%@", minutes))
-        if (seconds) components.push(this.formats.seconds.replace("%@", seconds))
+        if (days) components.push(evaluateFormat(this.formats.days, days))
+        if (hours) components.push(evaluateFormat(this.formats.hours, hours))
+        if (minutes) components.push(evaluateFormat(this.formats.minutes, minutes))
+        if (seconds) components.push(evaluateFormat(this.formats.seconds, seconds))
 
-        return <span className={styles.countdown}>{this.formattedTime(components, isNegative)}</span> 
+        return this.formattedTime(components, isNegative)
```

Comment:
Is there a reason for removing the wrapper tag here? (If you really mean to, please wrap this in a fragment.)